README.md

```diff
@@ -1,2 +1,42 @@
-# stellarcli
-A command line utility for accessing the StellarStation API
+# stellar
+
+A command line utility for accessing the StellarStation API. Let's build stellar apps together!
+
+## Using the app
+
+Precombiled binaries of the app can be found on the [releases page](https://github.com/infostellarinc/stellarcli/releases).
+
+We attempt to give detailed information on using the app in the tool's help message.
+
+```bash
+$ stellar -h
+```
+
+If you see anything is unclear, feel free to file an issue or send a pull request.
+
+## Authentication
+
+The utility can be authenticated using a StellarStation API key. If you don't have one yet,
+issue a key at your account page on StellarStation Console. The key can be activated using
+
+```bash
+$ stellar auth activate-api-key path/to/stellarstation-private-key.json
+```
+
+All commands after that will be authenticated using that key.
+
+## Developing
+
+The utility requires either Java or Go 1.11beta2. When using Java with Gradle, Go is automatically
```

Comment:
Can we make it work with non-beta version for go? The latest stable version is go1.10.

Reply:
go 1.11 adds module support, which improves developer experience of Go significantly. I don't want to start a new project in the old way, and 1.11 will be released this month.

Reply:
I see. It will be no problem if go 1.11 is released soon.